executor/analyze.go

```diff
@@ -558,6 +558,9 @@ func (e *AnalyzeColumnsExec) buildStats(ranges []*ranger.Range, needExtStats boo
 			if err != nil {
 				return nil, nil, nil, nil, err
 			}
+			if collectors[i].Samples[j].Value.Kind() == types.KindString {
+				collectors[i].Samples[j].Value.SetBytes(collectors[i].Samples[j].Value.GetBytes())
```

Comment:
What does this mean?

Reply:
+1, it would be helpful if you can add some comments here.